### ISSUE TITLE
TypeScript definition file doesn't export types

### ISSUE THREAD

**Issue by willyboy** [smilefam/SendBird-SDK-JavaScript#74]
Issue: The type definition file doesn't export types. Therefore, you can't use typing in your classes from the definition file. While autocomplete works, if say, I want to create a channel variable on my class and give it a channel type, I can't do that.

I played around with this a bit and I'm not sure how to do it with the "export = Sendbird" syntax (unless you have two separate type files). I can play more with it later but wanted to get this on the board.

**Comment by salguerooo** [comment]
I've been having the same issue, it's a bit annoying but we got around it by creating our own interfaces for channels/users. Ideally we should be abre to use something like `import { GroupChannel, GroupChannelListQuery } from 'sendbird'`.

**Comment by Alchemist85K** [comment]
@willyboy @salguerooo 
Thank you for your interest in SendBird SDK.
Currently it is not available for use as you said. I'll share this with the team and discuss it. 
When you use it in type script, please use as follows.
```
import * as SendBird from 'SendBird';
var sb = new SendBird({'appId': 'APP_ID'});
// sb.GroupChannel
```

**Comment by salguerooo** [comment]
This has been fixed on 3.0.56!
Thanks @Alchemist85K!
